webui/react/src/hooks/usePermissions.ts

```diff
@@ -372,10 +374,30 @@ const canViewExperimentArtifacts = (
 };
 
 // Model and ModelVersion actions
-// No permissions defined in PermissionType yet
-const canDeleteModel = ({ rbacAllPermission, user }: RbacOptsProps, model: ModelItem): boolean => {
-  // const permitted = relevantPermissions(userAssignments, userRoles);
-  return rbacAllPermission || (!!user && (user.isAdmin || user.id === model.userId));
+const canDeleteModel = (
+  { rbacAllPermission, rbacEnabled, rbacModel, user, userAssignments, userRoles }: RbacOptsProps,
+  model: ModelItem,
+): boolean => {
+  const permitted = relevantPermissions(userAssignments, userRoles, model.workspaceId);
+  return (
+    rbacAllPermission ||
+    (rbacEnabled && rbacModel
+      ? permitted.has(V1PermissionType.EDITMODELREGISTRY)
```

Comment:
question: will backend support `DELETEMODELREGISTRY` like workspace and experiment?

Reply:
I think that was intentionally not one of the permission types

So OK I think our final conclusion is to move an object, you need Modify on the origin, and Create on the destination. There is no Delete permission so we aren't going to use it.